### DISCUSSION TITLE
Welcome to ngx-unused-css Discussions!

### DISCUSSION THREAD

**Discussion by ivanblazevic** [ivanblazevic/ngx-unused-css#58] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eguesbaya** [comment]
Hi team, thank you for such a great project! I was really excited to try it on my Angular project, but unfortunately our style extension is .less, and therefore not covered by ngx-unused-css. Is this a feature you're currently working on?
